lacci/lib/shoes/drawables/para.rb

```diff
@@ -3,7 +3,7 @@
 class Shoes
   class Para < Shoes::Drawable 
   include FontHelper
-    shoes_styles :text_items, :size, :family, :font_weight, :font, :font_variant, :emphasis
+    shoes_styles :text_items, :size, :family, :font_weight, :font, :font_variant, :emphasis, :kerning
```

Comment:
@noahgibbs when we get these new excellent features; should we add them to manual.md you think?

Unless I'm being silly and kerning was an existing thing in manual.md 😅

Reply:
Kerning is in there already! We just only had it *work* recently :-)

There's a surprising amount of Shoes functionality we're still adding.